src/bun.js/bindings/NodeHTTP.cpp

```diff
@@ -922,12 +922,7 @@ static EncodedJSValue NodeHTTPServer__onRequest(
     args.append(thisValue);
 
     assignHeadersFromUWebSocketsForCall(request, methodString, args, globalObject, vm);
-    if (scope.exception()) {
-        auto* exception = scope.exception();
-        response->endWithoutBody();
-        scope.clearException();
-        return JSValue::encode(exception);
-    }
+    RETURN_IF_EXCEPTION(scope, {});
```

Comment:
is it okay to remove the `endWithoutBody` that used to be here?

Reply:
huh, great catch. im quite surprised this didn't break anything